### PR TITLE
webui

### PR DESCRIPTION
**Pull request by mfdeveloper508**
- refactoring with moov templater
- implemented webui, command line, web server

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ FROM scratch
 LABEL maintainer="Moov <oss@moov.io>"
 
 COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
-# COPY --from=builder /go/src/github.com/moov-io/fincen/bin/server /bin/server
+# COPY --from=builder /go/src/github.com/moov-io/server/bin/server /bin/server
```
Comment (adamdecaf) [inline Dockerfile:13]: This should be kept as fincen

File: cmd/webui/assets/index.html [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+        <h1>Fincen Report Parser</h3>
+            <p>
+                A go library for reading and writing Fincen BSA forms. It is capable of generating, validating, and batching submissions. Moov's <a href="https://github.com/moov-io/fincen">Fincen library</a>.
+                Paste an Fincen report on the left, in XML or JSON format, and the formatted version will be generated to the right.
+            </p>
```
Comment (adamdecaf) [inline cmd/webui/assets/index.html:101]: I was talking about a webui to create these form from `<input>` elements in HTML. A nicer builder / GUI for creating these. We can offer xml <-> json formatting, but that wasn't my main idea.

Comment (adamdecaf) [reply cmd/webui/assets/index.html:101]: I know that creating these forms with a bunch of `<input>` elements is a much larger task. Maybe we should use the `reflect` package to automatically generate the HTML?

Comment (mfdeveloper508) [reply cmd/webui/assets/index.html]: I removed unmeaningful parts, actually JSCON/XML convert button are unnecessary
this is new UI

error

user can verity the report form and generate xml attributes using this UI

Comment (adamdecaf) [reply cmd/webui/assets/index.html]: Okay that works for now. Thanks!

**Pull request merged**